geemusic/__init__.py

```diff
@@ -1,14 +1,21 @@
 from __future__ import absolute_import
 from flask import Flask
 from flask_ask import Ask
+from os import environ
 import logging
 
 from .utils.music import GMusicWrapper
 from .utils.music_queue import MusicQueue
 
 app = Flask(__name__)
 ask = Ask(app, '/alexa')
-logging.getLogger("flask_ask").setLevel(logging.INFO)
+
+if str(environ['DEBUG_MODE']) is True:
+    logLevel = logging.DEBUG
```

Comment:
nitpicking, but could you use `snake_case` for these variables to be consistent with the rest of the codebase?